src/app/server/config/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
We might want to check with people (Nim I guess) about whether a multilingual version might ever be wanted....

Reply:
Will do.